src/assets/images/in-darkBlue.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What's this "in-darkBlue"? Give it a better more clear name if you don't mind 🙏

Reply:
t's the linkedin icon xD I'll change it, thanks